docs/conventions/components.md

```diff
@@ -14,7 +14,6 @@ Please ensure the presence of the following attributes:
 
```

Comment:
in unpublish.js, there are some comments to update too 
  @ecl/vanilla-component-page-header
  @ecl/vanilla-component-breadcrumb
  
 in docs.md too
 
 You can search in whole project for @ecl/vanilla-